lk2nd/device/dts/msm8953/rules.mk

```diff
@@ -4,6 +4,7 @@ LOCAL_DIR := $(GET_LOCAL_DIR)
 ADTBS += \
 	$(LOCAL_DIR)/apq8053-lenovo-cd-18781y.dtb \
 	$(LOCAL_DIR)/msm8953-asus-zenfone3.dtb \
+	$(LOCAL_DIR)/msm8953-huawei-cannes.dtb  \
```

Comment:
Would you mind updating devices.md for this too?

Reply:
Ofc